packages/sql-graphql/lib/entity-to-type.js

```diff
@@ -50,10 +50,11 @@ function constructGraph (app, entity, opts, ignore) {
     /* istanbul ignore next */
     if (field.enum) {
       const enumValues = field.enum.reduce((acc, enumValue) => {
-        acc[enumValue] = { value: enumValue }
+        acc[enumValue.replace(/-/g, '_')] = { value: enumValue }
         return acc
       }, {})
-      meta.type = new graphql.GraphQLEnumType({ name: key, values: enumValues })
+      const enumName = `${entityName}_${key}`
+      meta.type = new graphql.GraphQLEnumType({ name: enumName, values: enumValues })
```

Comment:
Not sure why enums are still resolving to their names not values.

Reply:
I don't know, I'll take a look asap.